source/shared/cpp/ObjectModel/SharedAdaptiveCard.h

```diff
@@ -99,5 +102,6 @@ class AdaptiveCard
     std::vector<std::shared_ptr<BaseCardElement>> m_body;
     std::vector<std::shared_ptr<BaseActionElement>> m_actions;
 
+	std::shared_ptr<BaseActionElement> m_selectAction;
```

Comment:
>	 [](start = 0, length = 1)

Use spaces instead of a tab please.

Reply:
Sorry I thought I replaced all of them. Also thank you @almedina-ms for showing me how to set that up properly in VS as well. Fixed in 2a726b6